woocommerce/google-listings-and-ads
# Scheduled sync count

### Changes proposed in this Pull Request:

This PR adds the amount of scheduled jobs to the `product-statistics` endpoint. This can be used to show in the UI that a product sync is taking place.

Resolves part of #716

### Detailed test instructions:

1. In the Connection Test page create some async product sync requests, either individual products or all products
2. Send a request to `GET https://domain.test/wp-json/wc/gla/mc/product-statistics`
3. Confirm that the `scheduled_sync` is not 0
4. Wait for the scheduled syncs to complete and resend the request
5. Confirm that the `scheduled_sync` count is 0 again


### Changelog Note:
* Add - Scheduled sync count in the product statistics endpoint.
